dds/idl/marshal_generator.cpp

```diff
@@ -2646,28 +3106,81 @@ bool marshal_generator::gen_struct(AST_Structure* node,
         "    }\n";
 
       if (repr.xcdr1) {
-        be_global->impl_ <<
-          "    if (member_id == Serializer::pid_list_end";
+        be_global->impl_ << "    if (member_id == Serializer::pid_list_end";
         if (repr.not_only_xcdr1()) {
-          be_global->impl_ << " &&\n"
+          be_global->impl_ <<
+            " &&"
```

Comment:
The " &&" isn't starting a new line (in the output) so we don't need to wrap/indent here.